tutorials/strapi-app-serverless-containers-sqldb/index.mdx

```diff
@@ -0,0 +1,833 @@
+
+
+2. Access the folder you just created:
+    ```bash
+    cd my-blog
+    ```
+
+3. Install and save the ```node-postgres``` dependency:
+    ```bash
+    npm install pg --save
+    ```
+
+4. Run the following command to create a `Dockerfile`:
+    ```bash
+    touch Dockerfile
+    ```
+5. Add the following code to it:
+    ```bash
+    # Creating a multi-stage build
+    FROM node:20-alpine as build
+    RUN apk update && apk add --no-cache build-base gcc autoconf automake zlib-dev libpng-dev vips-dev git > /dev/null 2>&1
+    ARG NODE_ENV=development
+    ENV NODE_ENV=${NODE_ENV}
+
+    WORKDIR /opt/
+    #Copy package-lock.json only if it exists
+    COPY package.json package-lock.json* ./
+    RUN npm install -g node-gyp
+    RUN npm config set fetch-retry-maxtimeout 600000 -g && npm install
+    ENV PATH /opt/node_modules/.bin:$PATH
+    WORKDIR /opt/app
+    COPY . .
+    RUN npm run build
+
+    # Creating the final image
+    FROM node:20-alpine
+    RUN apk add --no-cache vips-dev
+    ARG NODE_ENV=development
+    ENV NODE_ENV=${NODE_ENV}
+    WORKDIR /opt/
+    COPY --from=build /opt/node_modules ./node_modules
+    WORKDIR /opt/app
+    COPY --from=build /opt/app ./
+    ENV PATH /opt/node_modules/.bin:$PATH
+
+
+    RUN chown -R node:node /opt/app
+    USER node
+    EXPOSE 1337
+
+    ENV ADMIN_EMAIL=admin@example.com
+    ENV ADMIN_FIRSTNAME=John
+    ENV ADMIN_LASTNAME=Doe
+
+    # Before starting the server, creates an admin user or updates its credentials if a user already exists with this email
+    # --silent option is used to avoid leaking credentials in logs.
+    CMD npm run --silent strapi admin:create-user -- --firstname=$ADMIN_FIRSTNAME --lastname=$ADMIN_LASTNAME --email=$ADMIN_EMAIL --password=$ADMIN_PASSWORD ; (npm run --silent strapi admin:reset-user-password -- --email=$ADMIN_EMAIL --password=$ADMIN_PASSWORD && npm run develop)
+    ```
+
+6. Create a `.dockerignore` file:
+    ```bash
+    touch .dockerignore
+    ```
+
+7. Add the following code to it:
+    ```bash
+    .tmp/
+    .cache/
+    .git/
+    .env
+    build/
+    node_modules/
+    backup/
+    ```
+
+    Docker will now ignore node dependencies and other unnecessary files during the build process.
+
+    At this stage, your application folder should be called `my-blog` and structured as follows:
+      ```bash
+      my-blog/
+      ├── build/
+      ├── config/
+      ├── data/
+      ├── database/
+      ├── node_modules/
+      ├── public/
+      ├── src/
+      ├── types/
+      ├── .dockerignore
+      ├── Dockerfile
+      ├── favicon.png
+      ├── jsconfig.json
+      ├── package.json
+      ├── README.md
+      └── yarn.lock					
+      ```
+
+8. Build your application container:
+    ```bash
+    docker build -t my-strapi-blog .   
+    ```
+    <Message type="tip">
+    The docker build image process can take a few minutes, particularly during `npm install` step since Strapi requires around 1 GB of node modules to be built.
```

Comment:
```suggestion
    The docker build image process can take a few minutes, particularly during the `npm install` step since Strapi requires around 1 GB of node modules to be built.
```